ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -331,6 +332,7 @@ public void handleNewConnection(final PeerConnection connection) {
     } catch (final PeerNotConnected peerNotConnected) {
       // Nothing to do.
     }
+    traceLambda(LOG, ethPeers.toString());
```

Comment:
probably this is not you want to achieve, to avoid calling the costly `ethPeers.toString()`, there are 2 options:

`traceLambda(LOG, "{}", ethPeers::toString);`

or more easily

`LOG.trace("{}", ethPeers");`

the same appliies to other similar statements in this PR

Reply:
fixed